### PR TITLE
Feature game c

### PR DESCRIPTION
**Pull request by Gian1080**

### CODE REVIEW

File: code/headers/HC_SR04_c.hpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+        /**
+     *  \brief
+     *  This class is for reading the data from the HC-SR04 ultrasone sensor
+     * 
+     *  \details
+     *   This class return the distance measured with the HC-SR04 ultrasone sensor, accurate up to 400 centimeters.
+     *   After 400 centimeters measurements are unuseable and should be disgarded.
+     */
+    class HC_SR04_c
```
Comment (ThePatman3) [inline code/headers/HC_SR04_c.hpp:14]: A bit strange that the *HC_SR04_c* class is added in a branch called *Feature game c* that also adds a *game_c* class.

File: code/remain.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#include <hwlib.hpp>
```
Comment (ThePatman3) [inline code/remain.cpp:1]: What is *remain.cpp* for?

**Review by ThePatman3 — COMMENTED** [review]
The code looks good, but I have left two comments I would rather have an answer on before I approve this pull request.

File: code/headers/HC_SR04_c.hpp (hunk 1 above)
Comment (Gian1080) [reply code/headers/HC_SR04_c.hpp:14]: I know, stranger things are happening

File: code/remain.cpp (hunk 2 above)
Comment (Gian1080) [reply code/remain.cpp:1]: to remain a cpp

**Pull request merged**